theano/scan_module/scan_opt.py

```diff
@@ -379,7 +407,10 @@ def process_node(self, fgraph, node):
 # This is a global opt for historical reason
 # It should be possible to change it to a local opt.
 class PushOutSeqScan(gof.Optimizer):
-
+    """
+    A global optimizer for pushing out the input variables that are not being
+    used inside the scan and provided in the sequences.
```

Comment:
Those variable are used by scan. What about this

    A global optimizer for pushing out some input sequences variables.